base-hack/src/misc/win_condition.c

```diff
@@ -27,17 +27,23 @@ void finalizeBeatGame(void) {
 // Pkmn Snap Spreadsheet: https://docs.google.com/spreadsheets/d/1nTZYi36dFaTB1XCgB7dJJffMsaKz-wOFmP5nDo8l3Uo/edit?usp=sharing
 
 static const short poke_snap_actors[] = {
+    117, // Beanstalk
```

Comment:
These are out of order with how you placed them in build.py. This will cause a difference between the two which is important when not all enemy types are placed